web/styles/left_sidebar.css

```diff
@@ -697,11 +699,11 @@ li.active-sub-filter {
         display: flex;
         align-items: center;
         justify-content: center;
-        /* TODO: Set this 24px height value as a variable.
-           Keep filter-icon height the full height of the box. */
-        height: 24px;
         /* Enlarge icons slightly in condensed views. */
-        font-size: 15px;
+        /* 15px at 16px/1em */
+        font-size: 0.9375em;
+        /* 24px at 15px/1em */
+        height: 1.6em;
```

Comment:
Is this 15px / 1em or 16px / em? The comment is not consistent with the code.

Reply:
I believe they are consistent: at the point where `font-size` is specified, an em is 16px. But as soon as we set `font-size` to the equivalent of 15px, that 15px em becomes the em for values like `height` elsewhere in the style block (and any of its child elements, too).